cli_FSD/utils.py

```
@@ -151,9 +151,9 @@ def use_mcp_tool(server_name: str, tool_name: str, arguments: dict) -> str:
         import json
         import subprocess
         from pathlib import Path
-        import os
         
         # Get MCP settings from config directory
+        from pathlib import Path
```

Ruff (0.8.2)

Redefinition of unused `Path` from line 153

Remove definition: `Path`

(F811)